themes/sexy-python/sexy-python.theme.bash

```diff
@@ -0,0 +1,46 @@
+# Small modification of Sexy Bash Prompt to include virtualenv and condaenv prompts
```

Comment:
For some reason, your last commit brought this file back. Looks like something went wrong with a merge...

Reply:
Thanks, I fixed this.